### ISSUE TITLE
Project cordova android decimal comma input does not work.

### ISSUE THREAD

**Issue by JoseManueIbnzLora** [dm4t2/vue-currency-input#126]
This is the instance of the component in my project cordova.
```
<currency-input
     class = "subtitle-1"
     v-model = "data"
     currency = "EUR"
     locale = "de"
     v-on: keyup.enter = "$ event.target.blur ()"
     style = "width: 100%;" />
```
When running the application on the Android device, the decimal point does not work on the keyboard and decimals cannot be entered.
I would appreciate any indication.
Thanks and great work.

**Comment by dm4t2** [comment]
This is related to #71. I now also think it is a good idea to be able to use both the dot and the comma as decimal symbols regardless of the configured locale. Otherwise it would be not possible to input a decimal symbol on mobile devices (see #88) if the browser locale is different to configured locale of the component. I have to look into it.

**Comment by dm4t2** [comment]
Fixed in [Release 1.20.0](https://github.com/dm4t2/vue-currency-input/releases/tag/1.20.0) 🎉 

Now you can use the decimal separators `.` (point), `,` (comma) and `٫` (Persian *momayyez*) independent from the current locale.
